src/pages/workspaces/workspace/Tools.js

```diff
@@ -269,11 +288,24 @@ export const Tools = _.flow(
     return _.find({ namespace, name }, configs)
   }
 
+  addRedactedAttribute(config, methods) {
```

Comment:
This function should be rewritten. What we want here is a function that computes whether a given config refers to a redacted method, and _returns_ that information. We also need to take into account that some method configs refer to Dockstore methods, which shouldn't get this treatment.